src/components/domain/survey/answerRow/TextRow.tsx

```diff
@@ -0,0 +1,34 @@
+import { UseFormRegister } from 'react-hook-form';
+import MaxLengthMultiTextArea from '@/components/common/input/MaxLengthMultiTextInput';
+import CirclePlanetIcon from '../../user/CirclePlanetIcon';
+
+interface TextRowProps {
+  name: string;
+  member: string;
+  register: UseFormRegister<Record<string, unknown>>;
+}
+
+export default function TextRow({ name, member, register }: TextRowProps) {
+  return (
+    <div className="mb-2 flex w-full rounded-[20px] bg-gray-100 py-2 md:px-8">
+      <div className="flex items-center gap-4">
+        <CirclePlanetIcon className="bg-gray-200" />
+        <span className="mr-4 mobile1">{member}</span>
+      </div>
+      <div className="flex w-[80%] flex-col space-y-3">
+        <MaxLengthMultiTextArea
+          maxLength={50}
+          placeholder="강점을 알려 주세요."
+          className="h-[60px] w-full max-w-[670px] border-2"
+          {...register(`${name}_short`)}
+        />
+        <MaxLengthMultiTextArea
+          maxLength={200}
+          placeholder="자세한 예를 들어 설명해강점을 알려 주세요."
+          className="h-[150px] w-full max-w-[670px] border-2"
```

Comment:
저희 200자를 다 채워도 기본 높이보다 낮기때문에 기본 높이를 더 낮게 만들어도 될거같아요
그리고 MaxLengthMultiTextArea가 현재 텍스트 높이에 맞춰 높이가 지정되어서, 넓었다가 한글자 쳤을 때 줄어드는게 좀 어색합니다! (동영상 참고)
낮게 시작해서 사용자가 글자를 치면 늘어나는게 더 괜찮을 것 같습니다!

```suggestion
          className="h-[60px] w-full max-w-[670px] border-2"
```

<img width="896" alt="image" src="https://github.com/user-attachments/assets/622a8c06-d4a0-4029-b8de-4ed933fd8e39">

https://github.com/user-attachments/assets/69705d99-0f4d-4a56-b417-fbe5c69a4067

Reply:
이 부분 저도 어색했는데 일단 피그마 GUI 기준으로 작업했습니다! 스크린 기준으로 변경하는게 나을 것 같네요,,

Reply:
✅ [a239fe4](https://github.com/PRism-Works/PRism-FE/pull/92/commits/a239fe4c9a467a2dab0b499a5303905fa9b5fb68)

Reply:
멘션 달아주신 커밋에 없어서 확인해보니 a239fe4c9a467a2dab0b499a5303905fa9b5fb68  여기였네요!